cla-backend-go/utils/gitlab.go

```diff
@@ -0,0 +1,121 @@
+// Copyright The Linux Foundation and each contributor to CommunityBridge.
+// SPDX-License-Identifier: MIT
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/communitybridge/easycla/cla-backend-go/config"
+	log "github.com/communitybridge/easycla/cla-backend-go/logging"
+	"github.com/sirupsen/logrus"
+	"github.com/xanzy/go-gitlab"
+)
+
+type GatedGitlabUser struct {
+	*gitlab.User
+	Err error
+}
+
+var missingCompanyAffiliation = errors.New("must confirm affiliation with their company")
+
+func PrepareMrCommentContent(missingUsers []*GatedGitlabUser, signedUsers []*gitlab.User, signURL string) string {
```

Comment:
You can maintain in the previous package. No need for utils package as it can get populated

Reply:
I am using this method on signatures->service.go, If i keep it on previous package , It will show cyclic dependency issue, as gitlab_activity is using signature package.